### PR TITLE
Added support for domain mode for messaging transactions tests

### PR DESCRIPTION
**Pull request by honza-kasik**

### CODE REVIEW

File: transactions/README.md [hunk 1]
```diff
@@ -11,7 +9,7 @@ When running this module, the server has to have already replaced its journal by
 
 You can do this either by
 
-* running `./prepare-and-start-server.sh -t <path to prepared journal> -u <path to unzipped EAP>`
+* running `./prepare-and-start-server.sh -t <path to prepared journal> -u <path to unzipped EAP>` with optional `-d` when running tests in domain mode
```
Comment (rhatlapa) [inline transactions/README.md]: does it mean that running with `-d` means testing server in domain mode? If yes, it should be better described as currently it sounds as some option which makes sense in domain mode only without any explanation what it does.

**Comment by rhatlapa** [comment]
Generally LGTM, please just update the description of the `-d` option

**Comment by honza-kasik** [comment]
@rhatlapa Thanks! I fixed description and added description to script itself.

File: transactions/README.md [hunk 2]
```diff
@@ -1,15 +1,18 @@
...
 
-When running this module, the server has to have already replaced its journal by the one from attached zip `artemis-journal.zip`.
+When running this module, the server has to have already replaced its journal by the one from attached zip 
+`artemis-journal.zip`.
 
 You can do this either by
 
-* running `./prepare-and-start-server.sh -t <path to prepared journal> -u <path to unzipped EAP>` with optional `-d` when running tests in domain mode
+* running `./prepare-and-start-server.sh -t <path to prepared journal> -u <path to unzipped EAP>` which also starts and
+unsecures server. Use optional `-d` to prepare and start server in domain mode instead of standalone.
```
Comment (zebrik) [inline transactions/README.md]: I would suggest to reword from  

> Use optional `-d` to prepare 

to something like 

> Use optional `-d` argument to prepare

**Review by zebrik — COMMENTED** [review]
Just one minor suggestion. I don't insist on it. If you squash I will merge.

**Comment by honza-kasik** [comment]
@zebrik Thanks! I updated description and squashed commits!

**Comment by zebrik** [comment]
Thank you! Merging..

**Pull request merged**